centinel/primitives/http.py

```diff
@@ -1,9 +1,12 @@
 import httplib
 import threading
 import time
+import urlparse
 
-def get_request(host, path="/", headers=None, ssl=False,
+# Actually gets the http. Moved this to it's own private method since it is called several times for following redirects
```

Comment:
It would be more Pythonic to put this comment into a doc string because then it can be auto-parsed for some documentation stuff. Doc strings are just strings placed immediately below the function definition, e.g.:

def something(args):
      """Here I am writing a function to demonstrate some stuff
          Isn't it cool that if I use 3 quotation marks I get a multiline doc string?

    """"
    # insert code here
    pass